BenchBnb/BenchBnb/Models/Review.cs

```diff
@@ -8,7 +10,16 @@ namespace BenchBnb.Models
     public class Review
     {
         public int Id { get; set; }
-        float Rating { get; set; }
-        string Comment { get; set; }
+        [Required, Column("Rating")]
+        public float Rating { get; set; }
```

Comment:
Again, I'd only use `Required` and `Column` attributes when necessary. The idea is to not write code that you don't need to write.

Reply:
Fixed I just kept them as required as a review requires a comment and rating

Reply:
Value types don't need `Required` attributes--they're non-nullable.